src/styles/app.css

```diff
@@ -663,13 +663,13 @@ h6 {
 /* ---------------------------------------------------------- */
 
 .site-head {
-    padding-top: 20px;
+    /* padding-top: 20px; */
     padding-bottom: 20px;
     color: #fff;
     background: var(--color-base);
     background-position: center;
-    background-size: cover;
-    background-image: url("../images/cook.jpg");
+    background-size: contain;
```

Comment:
Issue on mobile with repeated image, likely due to this change
![image](https://user-images.githubusercontent.com/35622595/125349786-d2c56380-e312-11eb-8a9e-ab486d46d56f.png)

Also on desktop for pages besides index 
![image](https://user-images.githubusercontent.com/35622595/125349899-f2f52280-e312-11eb-88d1-5b05da7c01e1.png)